### DISCUSSION TITLE
Like buttons plugin

### DISCUSSION THREAD

**Discussion by giuseppelt** [chrismwilliams/astro-theme-cactus#73] [Ideas]
Hi, I created an [astro plugin](https://www.feelback.dev) which allows to add *Like* buttons to blog posts.
Are you open to integrate it in your template? I can create a PR to setup the integration. It will be just a few line of code. I will make the integration _optional_, meaning that if the `setting_key` is missing (the user doesn't want to use the plugin), nothing is affected and will change.

In general, the plugin allows to receive any kind of feedback, but I think for a blogsite _likes_ are more appropriate. Nonetheless, it's very easy and customizable.

Let me know.
Thanks.

**Comment by chrismwilliams** [comment]
Hi @giuseppelt thanks for the suggestion. The intention of the starter is to be lightweight and minimal on both the dev and build side, and so I'll leave it up to the user to add/apply any additional feature for themselves.

Thanks again for the suggestion
